docs/guides/migrating_to_active_mesh_transit_network.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: "aviatrix"
+page_title: "Migrating from Classic Aviatrix Encrypted Transit Network to Aviatrix ActiveMesh Transit Network"
+description: |-
+Aviatrix Transit Network Migration Guide
```

Comment:
This is not valid YAML, need to add some spaces here. You can check the YAML validity here http://yaml-online-parser.appspot.com/
```suggestion
layout: "aviatrix"
page_title: "Migrating from Classic Aviatrix Encrypted Transit Network to Aviatrix ActiveMesh Transit Network"
description: |-
  Aviatrix Transit Network Migration Guide
```